src/net/rpc.rs

```diff
@@ -303,20 +307,90 @@ impl RPCPoxInfoData {
             .to_owned()
             .expect_u128() as u64;
 
-        let total_required = total_liquid_supply_ustx
-            .checked_div(rejection_fraction)
-            .expect("FATAL: unable to compute total_liquid_supply_ustx/current_rejection_votes");
+        let total_required = (total_liquid_supply_ustx as u128)
+            .checked_div(100)
+            .expect("FATAL: unable to compute total_liquid_supply_ustx * 100")
```

Comment:
I'm not sure the `expect` is necessary here?  `checked_div(INTEGER_CONSTANT)` I don't think can ever fail, so why not just use plain old integer divide?

Reply:
Actually, can we just make this entire `total_required` calculation just error out instead of runtime panic?  I see that we're already doing that below in the event of a numeric overflow.

Reply:
Sure

Reply:
Updated in 9e28b0a3851de7495c240c53595c65c477497033